themes/hamnportalen_tema/template-parts/pages/port/map.php

```diff
@@ -1 +1,33 @@
-<article class="port-map port-article">this is map</article>
+<?php
+  require_once(str_replace("/port", "/search-results/.env.php", __DIR__));
```

Comment:
Jag får ett felmeddelande när den här ska läsas in:
`: require_once(C:\xampp\htdocs\hamnportalen_new\wp-content\themes\hamnportalen_tema\template-parts\pages\port): Failed to open stream: Permission denied in
C:\xampp\htdocs\hamnportalen_new\wp-content\themes\hamnportalen_tema\template-parts\pages\port\map.php
on line
2
`

Reply:
Lite mer googlande löste problemet med denna rad istället:
`  require_once(realpath( dirname( __FILE__ ) ).'/../search-results/.env.php' );`

Reply:
Super, fixar direkt! Tack!

Reply:
Problemet beror troligtvis på att str_replace söker efter "/port", medan i felmeddelandet innehåller url:en "\port" vilket inte blir en match (gissar att det beror på hur windows vs mac hanterar url:er). 

Har bytt till en kortare variant av den du föreslog - slog bara ihop dirname(__FILE__) till __DIR__ vilket gör exakt samma sak.